.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*.js]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
According to https://editorconfig.org/
you have to download a plugin for Atom, Emacs, VSCode and Sublime to make this editorconfig working. I do not think this is possible to maintain this plugin across all the developers.
The eslinter itself should be able to conform to each of its formats.